exercises/practice/largest-series-product/runtests.jl

```diff
@@ -0,0 +1,65 @@
+using Test
+
+include("largest-series-product.jl")
+
+@testset "finds the largest product if span equals length" begin
+    @test largest_product("29", 2) == 18
+end
+
+@testset "can find the largest product of 2 with numbers in order" begin
+    @test largest_product("0123456789", 2) == 72
+end
+
+@testset "can find the largest product of 2" begin
+    @test largest_product("576802143", 2) == 48
+end
+
+@testset "can find the largest product of 3 with numbers in order" begin
+    @test largest_product("0123456789", 3) == 504
+end
+
+@testset "can find the largest product of 3" begin
+    @test largest_product("1027839564", 3) == 270
+end
+
+@testset "can find the largest product of 5 with numbers in order" begin
+    @test largest_product("0123456789", 5) == 15_120
+end
+
+@testset "can get the largest product of a big number" begin
+    @test largest_product("73167176531330624919225119674426574742355349194934", 6,) == 23_520
+end
+
+@testset "reports zero if the only digits are zero" begin
+    @test largest_product("0000", 2) == 0
+end
+
+@testset "reports zero if all spans include zero" begin
+    @test largest_product("99099", 3) == 0
+end
+
+@testset "reports 1 for empty string and empty product 0 span" begin
+    @test largest_product("", 0) == 1
+end
+
+@testset "reports 1 for nonempty string and empty product 0 span" begin
+    @test largest_product("123", 0) == 1
+end
+
+@testset "error handling" begin
```

Comment:
I don't know if it makes sense to use other test sets too?

Reply:
I think it's fine.